libbeat/cfgfile/list.go

```diff
@@ -101,8 +102,11 @@ func (r *RunnerList) Reload(configs []*reload.ConfigWithMeta) error {
 		r.logger.Debugf("Starting runner: %s", runner)
 		r.runners[hash] = runner
 		runner.Start()
+		moduleStarts.Add(1)
 	}
 
+	moduleRunning.Set(int64(len(r.runners)))
```

Comment:
Let's add a comment here that the metric is not necessarily correct. The Reload method calls `go runner.Stop` and immediately removes the runner from the hashtable. But in fact (especially filebeat) the runner/input might still be active for an unknown amount of time (it's a common problem with reconfigured filebeat inputs erroring out on Start).

Reply:
Good point, I guess the original name wasn't necessary the best for this. I added some comments.